### DISCUSSION TITLE
Macro Z_DEVICE_DT_DEV_NAME used in include/zephyr/drivers/smbus.h isn't define anywhere.

### DISCUSSION THREAD

**Discussion by KunLuffy** [zephyrproject-rtos/zephyr#69232] [General] [answered]
In macro SMBUS_DEVICE_DT_DEFINE, it use Z_DEVICE_DT_DEV_NAME but not defined.
Should It be changed to macro Z_DEVICE_DT_DEV_ID?

#define SMBUS_DEVICE_DT_DEFINE(node_id, init_fn, pm_device,		\
			       data_ptr, cfg_ptr, level, prio,		\
			       api_ptr, ...)				\
	Z_SMBUS_DEVICE_STATE_DEFINE(node_id,				\
				    Z_DEVICE_DT_DEV_NAME(node_id));	\
	Z_SMBUS_INIT_FN(Z_DEVICE_DT_DEV_NAME(node_id), init_fn)		\
	Z_DEVICE_DEFINE(node_id, Z_DEVICE_DT_DEV_NAME(node_id),		\
			DEVICE_DT_NAME(node_id),			\
			&UTIL_CAT(Z_DEVICE_DT_DEV_NAME(node_id), _init),\
			pm_device,					\
			data_ptr, cfg_ptr, level, prio,			\
			api_ptr,					\
			&(Z_DEVICE_STATE_NAME(Z_DEVICE_DT_DEV_NAME	\
					      (node_id)).devstate),	\
			__VA_ARGS__)

**Comment by yashi** [comment, marked as answer]
Right.
This one must have not worked at all.  https://github.com/zephyrproject-rtos/zephyr/commit/cfd3560e1b6cf2e647402934d83e7080c6046f13 is pre v3.3.  But https://github.com/zephyrproject-rtos/zephyr/commit/adff73d618bde2c83e9e6566ff199e27a79b7aad is post v3.3.
